doc/content/devices/adding-devices/_index.md

```diff
@@ -39,11 +39,15 @@ To use the device repository, make sure the **From the LoRaWAN Device Repository
 
 Choose a **Frequency plan** appropriate for your region. Your device and gateway must use the same frequency plan to communicate.
 
-Enter a **JoinEUI/AppEUI** if provided by your manufacturer. If your device is programmable, you may use all 0's, and then program the same JoinEUI/AppEUI in the device.
+Enter a **JoinEUI/AppEUI** if provided by your manufacturer. If your device is programmable, you may use the **Fill with zeros** button, and then program the same JoinEUI/AppEUI (`0000000000000000`) in the device.
 
-Enter your **DevEUI**.
+Enter your **DevEUI**. This should be provided by your manufacturer for commercial devices. If your device is programmable, you may generate an EUI using the **Generate** button, and program it in your device.
 
-If your manufacturer provides an **AppKey**, enter it. Otherwise, use the generate button to create one, and program it in to your device.
+For LoRaWAN version 1.0.x devices, you will see an **AppKey** field. If your manufacturer provides an **AppKey**, enter it. Otherwise, use the generate button to create one, and program it in to your device.
+
+For LoRaWAN version 1.1.x devices, you will also see a **NwkKey** field. If it is provided by your manufacturer, enter it. Otherwise, use the generate button to create one, and program it in to your device.
```

Comment:
```suggestion
For LoRaWAN version 1.1.x devices, you will also see a **NwkKey** field. If it is provided by your manufacturer, enter it. Otherwise, use the **Generate** button to create one, and program it in to your device.
```